db.go

```diff
@@ -258,18 +258,11 @@ func (s *ColumnStore) ReplayWALs(ctx context.Context) error {
 	for _, f := range files {
 		databaseName := f.Name()
 		g.Go(func() error {
-			db, err := s.DB(ctx, databaseName)
+			_, err := s.DB(ctx, databaseName)
```

Comment:
I left it in because it allows one to replay all the wals before the columnstore is considered ready. Otherwise whenever a write occurs on a db that hadn't been opened yet it would block that write to perform the replay.

Reply:
I'm not sure I understand. Shouldn't it be sufficient to move this code to `ColumnStore.New`? The columnstore can't be considered ready until that returns.

Reply:
Yea that's a good point, I'll move it there